test/e2e/test-dynamic-on.ts

```diff
@@ -74,7 +74,25 @@ module.exports = (testOrchestrator, testConfig) => {
     })
 
     const stats = await orchestrator.run()
+    t.equal(stats.successes, 1)
+    t.end()
+  })
 
+  test('dna registration', async t => {
```

Comment:
Can we check that one dna is initially installed and then after the `registerDna()`  it should return 2 dnas